helpers/calendar/ratios_and_utilization.py

```diff
@@ -72,13 +75,20 @@ def get_analytics_ratios_per_room(self, query, start, end):
                                                         app_events_list)
 
             ratio_object = RoomAnalyticsRatios().map_results_to_ratio_class(
-                                checkins,
-                                cancellations,
-                                calendar_events_list,
-                                app_events_list,
-                                room['name'])
+                checkins=checkins,
```

Comment:
Good work @jesseinit . What is the reason behind replacing the lines above and use an assignment like `checkins=checkins`?

Reply:
That's because the `map_results_to_ratio_class` takes in kwargs as params.
I was only trying to set various values to different keys on that part of the code.

Reply:
Okay, I get it.